Cargo.toml

```diff
@@ -73,11 +77,15 @@ surfman-chains-api = "0.2"
 thin-slice = "0.1.0"
 time = "0.1.41"
 tokio = "1"
+tokio-rustls = "0.24"
+tungstenite = "0.19"
 unicode-bidi = "0.3.4"
 unicode-script = "0.5"
 url = "2.0"
 uuid = { version = "1.3.4", features = ["v4"] }
 webdriver = "0.48.0"
+webpki-roots = "0.23"
+webpki = "0.22"
```

Comment:
Instead of using webpki 0.22 I'd suggest using rustls-webpki 0.101 here to match the version that rustls 0.21.5+ uses.

Reply:
This direct dependency is now removed.